model/task/task.go

```diff
@@ -296,6 +296,14 @@ type TestResult struct {
 	LogTestName string `json:"log_test_name" bson:"log_test_name"`
 }
 
+func (tr *TestResult) TestName() string {
```

Comment:
Can you add a doc comment what this mean? It's not clear at all to a casual reader what the difference is between a "test name", a "log test name", and  "test file".

Reply:
I actually need two separate functions (one for the log test name and one for the display test name). Added both with better names and documentation.